fractal_server/app/routes/api/v2/_aux_functions_tasks.py

```diff
@@ -226,11 +266,23 @@ async def _verify_non_duplication_user_constraint(
     res = await db.execute(stm)
     duplicate = res.scalars().all()
     if duplicate:
+        user = await db.get(UserOAuth, user_id)
+        if len(duplicate) > 1:
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail=(
+                    f"User '{user.email}' already owns {len(duplicate)} task "
+                    f"groups with name='{pkg_name}' and {version=}.\n"
+                    "Warning: this should have not happened, please contact an"
```

Comment:
This message should be more explicit.
The first part (User already owns..) is OK. The second part is also OK. What is missing is the fact that we have more than one item.

Let's change the second sentence into something like:
> Invalid state: found 3 matching task groups (IDs: [1, 2, 3]). This should have not happened, please contact an admin.